src/entities/mentor/ui/mentor-feedback-card/mentor-feedback-card.module.scss

```
@@ -0,0 +1,105 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  gap: 24px;
+}
+
+.card-wrapper {
+  max-width: 624px;
+  min-height: 510px;
+  padding: 40px 32px 40px 32px;
+
+  border-radius: 12px;
+  box-shadow: 0 4px 12px 0 rgb(0 0 0 / 10%);
```

```suggestion
  box-shadow: 0 4px 12px 0 hsla(from $color-black h s l/ $opacity-8);
```
Or we can add `$opacity-10` to our constants. But I'm sure we won't see the difference between 8% and 10%, so there's no point to multiply entities

---

fixed